# HG changeset patch
# User Tomas Mraz <tmraz@redhat.com>
# Date 1311330753 -7200
# Node ID f188e5c5fb249e5cf01f38e1ec129f3abeed5887
# Parent  bfd94f1e4dd0c0d7fecec86b998729e7a3050192
Drop the no longer touched files from documentation. Add new ones. (#708850)

diff -r bfd94f1e4dd0 -r f188e5c5fb24 man/en/authconfig.8
--- a/man/en/authconfig.8	Fri Jul 22 12:12:02 2011 +0200
+++ b/man/en/authconfig.8	Fri Jul 22 12:32:33 2011 +0200
@@ -1,7 +1,7 @@
 .de FN
 \fI\|\\$1\|\fP
 ..
-.TH AUTHCONFIG 8 "31 March 2010" "Red Hat, Inc."
+.TH AUTHCONFIG 8 "22 July 2011" "Red Hat, Inc."
 .SH NAME
 authconfig, authconfig-tui \- an interface for configuring system authentication resources
 .SH SYNOPSIS
@@ -111,7 +111,9 @@
 Currently includes variables named USESHADOW, USEMD5, USEKERBEROS, USELDAPAUTH,
 USESMBAUTH, USEWINBIND, USEWINBINDAUTH, USEHESIOD, USENIS, USELDAP, and others.
 .TP
-.FN /etc/passwd, /etc/shadow
+.FN /etc/passwd
+.TP
+.FN /etc/shadow
 Used for shadow password support.
 .TP
 .FN /etc/yp.conf
@@ -121,27 +123,33 @@
 Another configuration file for NIS support.
 .TP
 .FN /etc/ldap.conf
+.TP
+.FN /etc/nss_ldap.conf
+.TP
+.FN /etc/pam_ldap.conf
+.TP
+.FN /etc/nslcd.conf
+.TP
 .FN /etc/openldap/ldap.conf
-Used to configure LDAP (and OpenLDAP, respectively).
+Used to configure nss_ldap, pam_ldap, nslcd, and the OpenLDAP library. Only
+the files already existing on the system are modified.
 .TP
 .FN /etc/krb5.conf
 Used to configure Kerberos 5.
 .TP
-.FN /etc/krb.conf
-Used to configure Kerberos IV (write-only).
-.TP
 .FN /etc/hesiod.conf
 Used to configure Hesiod.
 .TP
-.FN /etc/pam_smb.conf
-Used to configure SMB authentication.
-.TP
 .FN /etc/samba/smb.conf
 Used to configure winbind authentication.
 .TP
 .FN /etc/nsswitch.conf
 Used to configure user information services.
 .TP
+.FN /etc/login.defs
+Used to configure parameters of user accounts (minimum UID of a regular user,
+password hashing algorithm).
+.TP
 .FN /etc/pam.d/system-auth
 Common PAM configuration for system services which include it using the
 \fBinclude\fR directive. It is created as symlink and not relinked if
@@ -156,7 +164,7 @@
 .PD
 .SH "SEE ALSO"
 authconfig-gtk(8), system-auth-ac(5), passwd(5), shadow(5), pwconv(1),
-domainname(1), ypbind(8), nsswitch.conf(5), smb.conf(5)
+domainname(1), ypbind(8), nsswitch.conf(5), smb.conf(5), sssd(8)
 
 .SH AUTHORS
 .nf
